src/View/mainpage/DrawROIWindow/UIDrawROIWindow.py

```diff
@@ -1132,6 +1130,45 @@ def onBoxDrawClicked(self):
         self.disable_cursor_diameter_transparency()
         self.has_drawing = False
 
+    def onClearClicked(self):
+        """
+        This function is used when reset button is clicked
+        """
+        self.dicom_view.image_display()
+        self.dicom_view.update_view()
+        self.isthmus_width_max_line_edit.setText("5")
+        self.internal_hole_max_line_edit.setText("9")
+        self.min_pixel_density_line_edit.setText("")
+        self.max_pixel_density_line_edit.setText("")
+        if hasattr(self, 'bounds_box_draw'):
+            delattr(self, 'bounds_box_draw')
+        if hasattr(self, 'drawingROI'):
+            delattr(self, 'drawingROI')
+            self.has_drawing = False
```

Comment:
as this line is the only difference in the two functions: **onClearClicked** and **onResetClicked**, you could possibly reduce the repetitive code here. For example, you could just call the reset function from the clear function